### ISSUE TITLE
Overwrite the system-config.toml file when ood-installer start

### ISSUE THREAD

**Issue by lizhihongTest** [buckyos/CYFS#196]
**Describe the bug**
After restarting the operating system, the ood-installer will be automatically launched by default, and the ood-installer will overwrite the system-config.toml and the service_version of the user-defined configuration

**To Reproduce**
```
[2023-04-11 20:31:59.361056 +08:00] WARN [ThreadId(1)] [tools\ood-installer\src\system_config_gen.rs:74] C:\cyfs\etc\ood-daemon\system-config.toml already exists! now will overwrite.
```

**Expected behavior**
If ood-installer finds that the system-config.toml file already exists, it should not overwrite it

**System information**
 ood-installer ：current version: 1.0.1.28-beta
system : Windows10

**Comment by weiqiushi** [comment]
The system-config.toml file is automatically generated at installation time and uses a different logic than the other configuration files. I will fix this soon

**Comment by weiqiushi** [comment]
you can use --overwrite flag on ood-installer to control system-config.toml overwrite logic:

with --overwrite: it will overwrite exists file.
default: if system-config.toml file already exists, skip overwrite it

**Comment by lurenpluto** [comment]
> you can use --overwrite flag on ood-installer to control system-config.toml overwrite logic:
> 
> with --overwrite: it will overwrite exists file. default: if system-config.toml file already exists, skip overwrite it

In the OOD-Installer, does this newly added parameter "--overwrite" also affect other configuration components, or is it only effective for the system-config.toml configuration? It seems that it could easily lead to some ambiguity.

**Comment by weiqiushi** [comment]
The parameter "--overwrite" was added a few months ago, it affects any configuration file released from asset and does not overwrite existing configuration files without the parameter.

Since system-config.toml is automatically generated from the code each time it is executed, it is not affected by "--overwrite" and will always overwrite the existing configuration file.

The change is to make the release logic of system-config.toml the same as the other configuration files, without "--overwrite", it will not overwrite the original file

**Comment by weiqiushi** [comment]
> > you can use --overwrite flag on ood-installer to control system-config.toml overwrite logic:
> > with --overwrite: it will overwrite exists file. default: if system-config.toml file already exists, skip overwrite it
> 
> In the OOD-Installer, does this newly added parameter "--overwrite" also affect other configuration components, or is it only effective for the system-config.toml configuration? It seems that it could easily lead to some ambiguity.

The parameter "--overwrite" was added a few months ago, it affects any configuration file released from asset and does not overwrite existing configuration files without the parameter.

Since system-config.toml is automatically generated from the code each time it is executed, it is not affected by "--overwrite" and will always overwrite the existing configuration file.

The change is to make the release logic of system-config.toml the same as the other configuration files, without "--overwrite", it will not overwrite the original file

**Comment by lizhihongTest** [comment]
This issue verifies that it has been fixed in version 1.1.0.527-nightly (23-04-13)
